.vscode/settings.json

```diff
@@ -6,7 +6,7 @@
 
     "editor.bracketPairColorization.enabled": true,
     "editor.inlayHints.enabled": "offUnlessPressed",
-    "editor.renderWhitespace": "all",
+    "editor.renderWhitespace": "none",
```

Comment:
Please revert this change

Reply:
bah you'd think "add to gitignore" would do something, but yeah git